test/fixtures/correct.js

```diff
@@ -202,6 +202,34 @@ const sinon = {};
 
 sinon.useFakeTimers({ toFake: ['Date'] });
 
+// `uphold-plugin/no-trailing-period-in-log-messages`
+const logger = {
+  debug: noop,
+  error: noop,
+  info: noop,
+  log: noop,
+  trace: noop,
+  warn: noop
+};
+
+logger.log('Valid log message without trailing period');
+logger.info('Processing complete');
+logger.warn('Warning message');
```

Comment:
The other tests also seem to say that `console.log` is picked up by the rule, maybe add some fixtures of it to the correct and incorrect?